additional note related to REST API env variables

martinpeck opened the pull request:
If you enable the Managed Service Identity within the Azure Portal, and then go straight to your Azure Function and run it, you may well find that `process.env['MSI_ENDPOINT']` and `process.env['MSI_SECRET']` return `undefined` because the process within which the Function is running does not have these environment variables set up.

Restarting the Function App, and "save and run" of the function, may also not bring these variables into the process.

The note warns users that this might be a problem.

azurecla commented:
Hi __@brohrer__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

rmca14 commented:
Sent to Gary for review.

garyericson commented:
This looks good.

martinpeck committed:
additional note related to REST API env variables

If you enable the Managed Service Identity within the Azure Portal, and then go straight to your Azure Function and run it, you may well find that `process.env['MSI_ENDPOINT']` and `process.env['MSI_SECRET']` return `undefined` because the process within which the Function is running does not have these environment variables set up.

Restarting the Function App, and "save and run" of the function, may also not bring these variables into the process.

The note warns users that this might be a problem.

martinpeck committed:
modify additional note related to REST API env variables

tynevi committed:
Update app-service-managed-service-identity.md
